src/user/tipsandtricks.rst

```diff
@@ -61,6 +61,15 @@ Here is how a ``.condarc`` file would look like:
 In addition to the channel priority we recommend to always install your packages inside a new environment instead the root environment from anaconda/miniconda.
 Using envs make it easier to debug problems with packages and ensure the stability of your root env.
 
+Note: in the past ``conda-forge`` used to vendorize some of ``defaults`` dependencies that were not built in our infrastructure,
+like compilers run-times, to avoid the mixing channel problem.
+However, with the ``strict`` option, we no longer have to vendorize those (which has its own problems too),
```

Comment:
I personally would leave out `(which has its own problems too)` but I guess that's a matter of taste.

Reply:
```suggestion
However, with the ``strict`` option, we no longer have to vendorize those (this led to its own set of problems),
```
@croth1 it seems like a good idea to indicate why these channels are marked as `broken` and effectively not available.  My preference would be for some brief explanation of what those problems where but it seems good to at least indicate that there was a good reason not to keep them available, given that it's caused some inconvenience for people like me that where told to use only the `conda-forge` channel before the `strict` option was available.

Reply:
Maybe a better explanation for this should be in the `News` section?

Reply:
Sure, that's fine.